src/test/java/org/sagebionetworks/bridge/udd/helper/SesHelperTest.java

```diff
@@ -89,6 +106,56 @@ public void testSendPresignedUrl() throws Exception {
         assertTrue(textEmail.contains(dummyExpirationDateStr));
     }
 
+    @Test
+    public void testSendEmailWithAttachmentToAccount() throws MessagingException, IOException {
+        // test attachment input by creating a temporary file to attach
+        FileHelper fileHelper = new FileHelper();
+        File tmpDir = null;
+        File tmpFile = null;
+        try {
+            tmpDir = fileHelper.createTempDir();
+            tmpFile = fileHelper.newFile(tmpDir, "testFile.txt");
+            CSVWriter csvWriter = new CSVWriter(fileHelper.getWriter(tmpFile));
+            csvWriter.writeNext("test text");
+
+            // execute
+            sesHelper.sendEmailWithAttachmentToAccount(appInfo, accountInfo, tmpFile.getAbsolutePath());
+
+            SendRawEmailRequest rawSesRequest = sesRawRequestCaptor.getValue();
+            RawMessage rawMessage = rawSesRequest.getRawMessage();
+
+            assertEquals(rawMessage.getData().get(0), 'F');
+            assertEquals(rawMessage.getData().get(1), 'r');
+            assertEquals(rawMessage.getData().get(2), 'o');
+            assertEquals(rawMessage.getData().get(3), 'm');
+            assertEquals(rawMessage.getData().get(4), ':');
+            assertEquals(rawMessage.getData().get(5), ' ');
+            assertEquals(rawMessage.getData().get(6), 's');
+            assertEquals(rawMessage.getData().get(7), 'u');
+            assertEquals(rawMessage.getData().get(8), 'p');
+            assertEquals(rawMessage.getData().get(9), 'p');
+            assertEquals(rawMessage.getData().get(10), 'o');
+            assertEquals(rawMessage.getData().get(11), 'r');
+            assertEquals(rawMessage.getData().get(12), 't');
+            assertEquals(rawMessage.getData().get(13), '@');
+            assertEquals(rawMessage.getData().get(14), 's');
+            assertEquals(rawMessage.getData().get(15), 'a');
+            assertEquals(rawMessage.getData().get(16), 'g');
+            assertEquals(rawMessage.getData().get(17), 'e');
```

Comment:
You can probably do new String(rawMessage.getData().array()) to convert this into a String.

Reply:
For some reason .toString() converts into "java.nio.HeapByteBuffer[pos=0 lim=1196 cap=1196]", but I found a different way of doing this.